app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, distance: "Distance") -> object:
```

Comment:
This function returns `Distance`. Fix type hint for return type everywhere
```suggestion
    def __add__(self, distance: "Distance") ->Distance:
```